.github/workflows/test.yml

```diff
@@ -90,7 +90,7 @@ jobs:
           project_id: '${{ vars.PROJECT_ID }}'
           workload_identity_provider: '${{ vars.WIF_PROVIDER_NAME }}'
 
-      - uses: 'google-github-actions/setup-gcloud@main'
+      - uses: 'google-github-actions/setup-gcloud@v2'
```

Comment:
Do we want to do this? Unless I missed something, this wont test the code in the pull request rather, whatever is tagged as v2. So anything that gets merged would be untested because we dont tag until release.

Reply:
@sethvargo

Reply:
disregard